static/js/publisher/metrics/graphs/activeDevices.js

```diff
@@ -58,6 +59,11 @@ const labelPosition = function(graphEl, labelText, points) {
     return;
   }
 
+  const activePoints = points.slice(range[0], range[range.length - 1]);
+  const topActiveItem = Math.max.apply(Math, activePoints);
+  const topItem = Math.max.apply(Math, points);
+  let y = bbox.y + (((bbox.height / topItem) * topActiveItem) / 2);
```

Comment:
This is only ever going to work for any version that is more than twice as popular as all newer versions put together. Which, for most snaps, means only the newest version.

For any version X where newer versions combined are more than half as popular, they will ascend to more than half the height of X’s area, so they’ll obscure or completely X’s halfway-down label.